### DISCUSSION TITLE
How do I make my components compatible with Chrome kernel version 85?

### DISCUSSION THREAD

**Discussion by Bamboo742** [mantinedev/mantine#6891] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

^7.12.1

### What package has an issue?

@mantine/core

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

All

### Describe the bug

I want to display it in lower version browsers, but I'm already using v7 and don't know how to roll back to v6
![image](https://github.com/user-attachments/assets/3339e291-4e33-4e0c-91cd-5b35aa42dfab)

![image](https://github.com/user-attachments/assets/be122ebf-bea3-4d3a-9a58-3bb7098e4421)

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
To use the previous version of the library, change packages version in the `package.json`file to the required version.
